testgen/src/main/groovy/GenerateApiDocsTest.groovy

```diff
@@ -0,0 +1,54 @@
+package com.vivy.springdoc
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+
+class GenerateApiDocsTest extends DefaultTask {
+
+    @Input
+    String openApiDocsTestOut;
+    @Input
+    String testPackage;
+    @Input
+    @Optional
+    String extensionClassImport = null;
+    @Input
+    @Optional
+    String extensionClassName = null;
```

Comment:
don't think we should configure it at all

Reply:
Yes, we can take it from `extensionClassImport`.